maistra/run-ci.sh

```diff
@@ -14,6 +14,11 @@ time bazel build \
 echo "Build succeeded. Binary generated:"
 bazel-bin/source/exe/envoy-static --version
 
+# Build tests
+time bazel build \
```

Comment:
Can we add a doc comment to explain why we do split the build and test steps for future readers?